### PR TITLE
192 support ubipools

### PR DESCRIPTION
**Pull request by sirpy**
# Description

- update subgraph link to one with ubipools
- add support for dev/qa G$ tokens so dev/qa pools can be tested

### CODE REVIEW

File: packages/app/src/components/TransactionList/SupportTransactionListItem.tsx [hunk 1]
```diff
@@ -31,6 +31,10 @@ export function SupportTransactionListItem({ transaction }: SupportTransactionLi
   const { data: ensName } = useEnsName({ address: userAddress, chainId: 1 });
   const userFullName = useFetchFullName(userAddress);
   const userIdentifier = userFullName ?? ensName ?? formatAddress(userAddress);
+  const flowUpdateLog = useWaitForTransaction({ hash: hash as `0x${string}`, chainId: 42220 });
+  const flowLogIndex = flowUpdateLog.data?.logs.find(
+    (_) => _.topics[0] === '0x57269d2ebcccecdcc0d9d2c0a0b80ead95f344e28ec20f50f709811f209d4e0e'
```
Comment (L03TJ3) [inline packages/app/src/components/TransactionList/SupportTransactionListItem.tsx:36]: can it be documented either in readme docs or code comment what the hash relates to?

File: packages/app/src/hooks/useTokenList.ts [hunk 2]
```diff
@@ -4,7 +4,10 @@ import { Token } from '@uniswap/sdk-core';
 import CeloTokenList from '../models/CeloTokenList.json';
 import { SupportedNetwork } from '../models/constants';
 
-const populatedTokenList: Record<string, Token> = {};
+const populatedTokenList: Record<string, Token> = {
+  'G$-Dev': new Token(42220, '0xFa51eFDc0910CCdA91732e6806912Fa12e2FD475', 18, 'G$-Dev'),
```
Comment (L03TJ3) [inline packages/app/src/hooks/useTokenList.ts:8]: should the addresses not be read from deployment.json?

File: packages/app/src/components/DonateComponent.tsx [hunk 3]
```diff
@@ -51,26 +51,39 @@ function DonateComponent({ collective }: DonateComponentProps) {
...
   const [duration, setDuration] = useState(12);
   const [decimalDonationAmount, setDecimalDonationAmount] = useState(0);
 
   const tokenList = useTokenList();
+  const gdEnvSymbol =
+    Object.keys(tokenList).find((key) => {
+      if (key.startsWith('G$')) {
+        return tokenList[key].address.toLowerCase() === collective.rewardToken.toLowerCase();
+      } else return false;
+    }) || 'G$';
```
Comment (L03TJ3) [inline packages/app/src/components/DonateComponent.tsx:64]:
```suggestion
  const gdEnvSymbol = Object.keys(tokenList).reduce((acc, key) => {
  return key.startsWith('G$') && tokenList[key].address.toLowerCase() === collective.rewardToken.toLowerCase()
    ? key
    : acc;
}, 'G$');
```

Reduce for readability

File: packages/app/src/components/DonateComponent.tsx [hunk 4]
```diff
@@ -51,26 +51,39 @@ function DonateComponent({ collective }: DonateComponentProps) {
...
-    let options = Object.keys(tokenList).map((key) => ({
-      value: key,
-      label: key,
-    }));
+    let options = Object.keys(tokenList)
+      .filter((key) => key.startsWith('G$') === false || key === gdEnvSymbol)
+      .map((key) => ({
+        value: key,
+        label: key,
+      }));
```
Comment (L03TJ3) [inline packages/app/src/components/DonateComponent.tsx:74]:
```suggestion
    let options = Object.keys(tokenList).reduce((acc, key) => {
  if (!key.startsWith('G$') || key === gdEnvSymbol) {
    acc.push({ value: key, label: key });
  }
  return acc;
}, []);
```

Reduce I find personally more readable.
and it also only requires a single loop

File: packages/app/src/components/FlowingDonationsRowItem.tsx [hunk 5]
```diff
@@ -2,32 +2,43 @@ import { Image, Text, View, StyleSheet } from 'react-native';
 import { InterRegular, InterSemiBold } from '../utils/webFonts';
 import { Colors } from '../utils/colors';
 import { useMediaQuery } from 'native-base';
-import { useDonorCollectivesFlowingBalances } from '../hooks/useFlowingBalance';
+import { useDonorCollectivesFlowingBalancesWithAltStaticBalance } from '../hooks/useFlowingBalance';
```
[the remark this answers is not in the document]
Comment (sirpy) [reply packages/app/src/components/FlowingDonationsRowItem.tsx:5]: dono, i just renamed the used file to the unused file. not my changes. but previous code.

File: packages/app/src/hooks/useTokenList.ts (hunk 2 above)
Comment (sirpy) [reply packages/app/src/hooks/useTokenList.ts:8]: fixed

File: packages/app/src/components/TransactionList/SupportTransactionListItem.tsx (hunk 1 above)
Comment (sirpy) [reply packages/app/src/components/TransactionList/SupportTransactionListItem.tsx:36]: done

**Review by L03TJ3 — APPROVED** [review]

**Pull request merged**